ujt/converters.py

```diff
@@ -75,3 +91,86 @@ def cytoscape_elements_from_clients(client_name_message_map: Dict[str, Client]):
                 })
 
     return node_elements + edge_elements
+
+
+def datatable_from_nodes(nodes, use_relative_names, table_id):
```

Comment:
Here are new functions to generate the Dash datatables that display information about nodes.